content/v2.0/introduction/getting-started.md

```diff
@@ -58,25 +57,27 @@ To create a Hanami application, you will need Ruby 3.0 or greater. Check your ru
 ruby --version
 ```
 
-If you need to install Ruby, follow with the instructions on [rubylang.org](https://www.ruby-lang.org/en/documentation/installation/).
+If you need to install or upgrade Ruby, follow the instructions on [rubylang.org](https://www.ruby-lang.org/en/documentation/installation/).
 
 ### Installing the gem
 
 In order to create a Hanami application, first install the hanami gem:
 
 ```shell
-gem install hanami --pre
+gem install hanami
 ```
 
-### Generating your first application
+### Using the hanami new command
```

Comment:
This reads a bit weird IMO. Can "hanami new" be in backticks, so it's clear that it is the complete command?

Reply:
Thanks @katafrakt - I've changed it to "Using the application generator". I did try the backticks but the formatting doesn't look great in a heading.

Reply:
nice, that's probably even better heading